content/articles/rocket-lander-in-elm.md

```diff
@@ -0,0 +1,198 @@
+Title: Experience report: Rocket lander in Elm
+Date: 2015-02-15
+Short_summary: We build a small game in the programming language Elm.
+Category: Dev
+Authors: Tom
+
+*Reading time: ~10 minutes.*
+
+[Play the demo (desktop only, sorry!)](/rocket-lander-in-elm-extra/ship.html)
+
+In a comedy contest for the funniest language alive JavaScript would
+win hands-down. For that reason we're investigating various new
```

Comment:
I don't know, it's the nicest thing I can say about JS.. And it can be read as tongue-in-cheek... How would you phrase it?